open-media-match/src/OpenMediaMatch/storage/interface.py

```diff
@@ -267,12 +267,15 @@ def get_bank(self, name: str) -> t.Optional[BankConfig]:
         return self.get_banks().get(name)
 
     @abc.abstractmethod
-    def bank_update(self, bank: BankConfig, *, create: bool = False) -> None:
+    def bank_update(
+        self, bank: BankConfig, *, create: bool = False, name: t.Optional[str] = None
+    ) -> None:
         """
         Update a bank config in the backing store.
 
         If create is false, will throw an exception if not already existing.
         If create is true, will throw an exception if it already exists
+        If create is false, name has to be provided
```

Comment:
blocking: name is only required if you are changing the name. 

Suggest changing "name" to "rename_from", looking up the old record with bank.name unless rename is true.

Reply:
nit: Comment needs updating